apollo-openapi/src/test/java/com/ctrip/framework/apollo/openapi/client/service/ItemOpenApiServiceTest.java

```diff
@@ -75,6 +76,26 @@ public void testGetItem() throws Exception {
             someAppId, someCluster, someNamespace, someKey), get.getURI().toString());
   }
 
+  @Test
+  public void testGetItemByIllegalKey() throws Exception {
+    String someKey = "/";
```

Comment:
It's better to use a more meaningful key, e.g.
```suggestion
    String someKey = "a/b";
```

Reply:
ok.